views/js/runner/plugins/security/templates/watermarkCirclePath.tpl

```diff
@@ -0,0 +1,6 @@
+<svg width="{{boxSize}}" height="{{boxSize}}" viewBox="0 0 {{boxSize}} {{boxSize}}" xmlns="http://www.w3.org/2000/svg">
+    <!-- <defs> -->
+        <path id="circlePath" fill="none" stroke="none" d="M {{s}},{{f}} A {{r}},{{r}} 0 1,1 {{s}},{{sr}} A {{r}},{{r}} 0 1,1 {{s}},{{f}}" />
+    <!-- </defs> -->
+    <text><textPath href="#circlePath">{{repeatedText}}</textPath></text>
+</svg>
```

Comment:
When applying config `type: 'circle', style: 'color: blue;'` , blue is not applied. I think it can be fixed by attribute or CSS: `fill: currentColor`.